web/src/pages/index.page.tsx

```diff
@@ -1,8 +1,12 @@
 import { ArrowDownward } from "@mui/icons-material";
 import { Box, Button, Divider, Typography } from "@mui/material";
+// import Image from "material-ui-image";
+// Cannot find module '@material-ui/core/CircularProgress' --> error I get when I try to use the above package --> does it really have dependency on CircularProgress
```

Comment:
Just a general comment on this commit - I was experimenting with a few different things and didn't think you'd look at it until I marked it ready for review. So that probably explains the unfinished/confusing state of this. Nonetheless, I appreciate the feedback. :)

I was messing around with `material-ui-image` before I saw a ESLint warning suggesting that I use Next's Image component.

Reply:
allowing the base URL via next.config.js should be fine. storing the image in the repo will not work if the image is trying to be retrieved from the client (if you look at other images currently stored in the repo, they're stored under `public/`, so they're exposed to the web and can be accessed from the client)